@xen-orchestra/lite/src/components/Console.tsx

```diff
@@ -14,9 +15,10 @@ interface ParentState {
 }
 
 interface State {
-  container: React.RefObject<HTMLDivElement>
+  // Type error with HTMLDivElement.
+  // See https://github.com/styled-components/styled-components/issues/2202
```

Comment:
The issue has been closed in favor of [this one](https://github.com/DefinitelyTyped/DefinitelyTyped/issues/30451).